python/src/trezorlib/messages.py

```
@@ -1136,6 +1170,148 @@
         self.application = application
 
 
+class BenfenGetAddress(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = 12201
+    FIELDS = {
+        1: protobuf.Field("address_n", "uint32", repeated=True, required=False, default=None),
+        2: protobuf.Field("show_display", "bool", repeated=False, required=False, default=None),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)